hieradata/vagrant/common.yaml

```diff
@@ -254,21 +254,16 @@ profile::jenkinscontroller::jcasc:
             labels:
               - maven-21
               - jdk21
+              - node
+              - webbuilder
+              - ruby
             cpus: 4
             memory: 8
             imagePullSecrets: dockerhub-credential
             additionalPVCs:
               - claimName: ci-jenkins-io-maven-cache-readwritemany
                 mountPath: "/cache-rw"
                 readOnly: false
-          - name: jnlp-webbuilder
-            agentJavaBin: java
-            cpus: 2
-            memory: 4
-            labels:
-              - node
-              - webbuilder
-              - ruby
             imagePullSecrets: dockerhub-credential
```

Comment:
FTR, this line should also have been removed. it made the YAML file non parseable (at least by `updatecli`):

```
ERROR: something went wrong in "target#updateVagrantCommonBuildAgentAwsCliVersion" : updating yaml file: parsing yaml file: [267:13] mapping key "imagePullSecrets" already defined at [262:13]
  264 |               - claimName: ci-jenkins-io-maven-cache-readwritemany
  265 |                 mountPath: "/cache-rw"
  266 |                 readOnly: false
> 267 |             imagePullSecrets: dockerhub-credential
                    ^
  268 |       second-cluster:
  269 |         enabled: true
  270 |         provider: azure
```

Reply:
Hot-fixed: https://github.com/jenkins-infra/jenkins-infra/commit/25a2174b1636dd77927327c98ad49d9fddf2f882